tests/integration/ceph_smoke_test.go

```diff
@@ -93,7 +93,6 @@ func (s *SmokeSuite) SetupSuite() {
 		ConnectionsEncrypted:      true,
 		ConnectionsCompressed:     true,
 		UseCrashPruner:            true,
-		EnableVolumeReplication:   true,
```

Comment:
Could we enable the CSI addons in the tests instead of completely removing this option? That might be a separate PR though.